src/system/utils.ts

```diff
@@ -14,47 +14,35 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
- // notice_end
+// notice_end
 
-export function removeAll(arr, item) {
-    for(let i = arr.length; i--;) {
-        if(arr[i] === item) {
+export function removeAll(arr: any[], item: any) {
```

Comment:
better to make this `removeAll<T>` and type `arr` and `item` as the same types

Reply:
fixed